sdcm/cluster.py

```diff
@@ -390,6 +397,10 @@ def __init__(self, name, parent_cluster, ssh_login_info=None, base_logdir=None,
         self._cassandra_stress_version = None
         self.lock = threading.Lock()
 
+        if IP_SSH_CONNECTIONS == 'public' or Setup.MULTI_REGION:
+            from sdcm.auto_ssh import start_auto_ssh
+            start_auto_ssh(Setup.test_id(), self, Setup.RSYSLOG_ADDRESS[1], '5000')
```

Comment:
use `SSH_TUNNEL_PORT` from the comment above as a default in the `start_auto_ssh` func

Reply:
done